src/components/Countdown/Countdown.js

```diff
@@ -0,0 +1,78 @@
+import styled from 'styled-components/macro'
+import { ReactComponent as Play } from '../../assets/play-icon.svg'
+import { ReactComponent as Stop } from '../../assets/stop-icon.svg'
+import { useState, useEffect } from 'react'
+
+export default function Countdown({ startMinutes = 25, startSeconds = 0 }) {
+  const playIcon = <Play />
+  const stopIcon = <Stop />
+
+  const [isActive, setIsActive] = useState(false)
+  const [timerExpired, setTimerExpired] = useState(false)
+  const [[minutes, seconds], setCounter] = useState([
+    startMinutes,
+    startSeconds,
+  ])
```

Comment:
nice, didn't know that this is possible in that way 🤙 kind of destructuring assignment, isn't it?